src/common/objects/CircleRigidBody.hpp

```diff
@@ -39,6 +39,10 @@ class CircleRigidBody : public GameObject {
         m_position = position;
     }
 
+    void interpolatePosition(const sf::Vector2f position, float a) {
+        m_position = VectorUtil::lerp(m_position, position, a);
```

Comment:
should this not be using hermite interpolation? why not?

Reply:
This is linear interpolation
I should update the name

We use hermite but i left this in there in case we ever want to do linear interpolation